.prettierignore

```diff
@@ -9,4 +9,5 @@ dist-start/
 dist-storybook/
 report/
 template/
-.changeset/*.md
\ No newline at end of file
+.changeset/*.md
+CHANGELOG.md
```

Comment:
Whats the reason for ignoring this?

Reply:
It's auto-generated by changesets and it breaks our rules.